source/models/index.md

```diff
@@ -1,162 +1,125 @@
+write a custom serializer. The serializer has two jobs - formatting your
+outgoing requests to match what your server expects, and processing the response
+so that it can be matched up correctly with an Ember Data Model. Similar to
+adapters, you can use one of Ember's out of the box options, find a community
+serializer, or write your own.
+
+Both adapters and serializers can be customized for each kind of Model, but
+most new apps can just define one application-wide adapter that is used for
+all record types. These tools let you completely change how your API is
+implemented without impacting your front end Ember application code.
+
+#### Making Server Requests
+
+Now you're ready to start making requests. If you have done front end
+development before, you may be familiar with writing requests by hand, on an
+as-needed and repeated basis... but here, Ember Data does all the heavy lifting.
+All you need to do is call a few special methods (like `findAll`, `findRecord`,
+`save`, and `destroy`) and the requests will be constructed for you, using
+information from your Models, Serializer, and Adapter.
+
+#### Loading the Server Response
+
+When response is received from the back end server, it is automatically matched up with a
+Model and saved in the Store. The store is a local cache of data that is kept
+up to date as you make more requests. This means that anywhere in your
+app that you need to reference or display data, you're looking at a single
+source of truth. It's like having a mini, temporary database available in the
+browser. Ember Data is smart enough that if you have redundant requests
+throughout your app, it will look in the Store first for the records, and
+save you an unnecessary server request. This is one of the ways that Ember
+apps achieve a speedy, smooth interface.
+
+#### Benefits to the developer
+
+All of these tools combined lead to fewer lines of code, high maintainability
+and extensibility, and an easy pathway for supporting changes made to your back end.
+
+## Frequently Asked Questions
+
+## Can I use Ember Data with my rare and special back end server/database?
+
+Yes, with some work on your part to write your own serializers and adapters.
+One of the goals of Ember Data is to make it possible to iterate on the back
+end without requiring a rewrite of the front end. For example, if a
+well-constructed Ember app is making its requests to a legacy back end server,
+switching over to a new server API could be as simple as writing some new
+adapters and serializers. This is one of the reasons why Ember is a good
+choice for long-lived or complex applications.
+
+## Do I need to use Ember Data in my Ember app?
+No. If you prefer to manage requests yourself, you can. Some developers pick
+and choose which Ember Data tools to utilize, such as fetching information
+manually but still loading it into the Store.
+
+## Why is JSON API the default?
```

Comment:
@jenweber Worth mentioning that JSON API and Ember Data have helped each other grow. Not sure any other JSON API specification provides the surface that Ember Data needs in order for some key abstractions to work (like magic). Key abstractions such as side-loading, loading relationships (`post.get('comments')`)